src/plob_codec.erl

```diff
@@ -41,7 +42,7 @@ decode({_, Decoder}, Val) -> decode2(Decoder, Val);
 decode(Decoder, Val) -> decode2(Decoder, Val).
 
 -spec decode2(decoder(), any()) -> any().
-decode2(_Codec, null) -> null;
+decode2(_Codec, null) -> undefined;
```

Comment:
Won't this break things?

Reply:
Quite possibly. If it breaks tests I'll fix them.

Reply:
I've removed this change again for now, given the tight schedule.